ISSUE_TEMPLATE.md

```diff
@@ -1 +1,47 @@
+<!--
+Please use the below format to report the issue in a better way.
+To check any option, replace the space between the "[" and "]" with a "x".
+-->
 
+#### Nature of Issue ?
+
+- [ ] Found a Bug
+- [ ] Existing Feature Enhancement
+- [ ] New Feature Request
+
+#### Most Appropriate Sub Area ?
+
+- [ ] Color
+- [ ] Core
+- [ ] Events
+- [ ] Image
+- [ ] IO
+- [ ] Math
+- [ ] Typography
+- [ ] Utilities
+- [ ] WebGL
+
+<!--
+	Fill the following sections as suitable, and delete the part which is not applicable.
+-->
+
+<!-- If you found a bug, then please consider the following points -->
+#### Details about the Bug : 
+
+- p5.js Version : 
+- Web Browser and it's Version :
+- Operating System :
+- Platform / Device : 
+- How can we reproduce this ? 
+<!--
```

Comment:
let's ask for a code snippet rather than a github repo, so we lower the chance of people posting large projects with hard to isolate bugs. ideally, the snippet is small enough to fit in the issue itself. a gist or jsfiddle or similar works too.

i think these posts are nice references for how to describe what we're asking for.
https://github.com/d3/d3/issues/new
https://gist.github.com/mbostock/370b737ce71bed0749e103b01ce1bfaf